package.json

```
@@ -123,6 +123,7 @@
   },
   "license": "GPL-3.0-or-later",
   "dependencies": {
-    "@gnosis.pm/gp-v2-contracts": "^0.0.1-alpha.10"
+    "@gnosis.pm/gp-v2-contracts": "^0.0.1-alpha.10",
+    "@uniswap/sdk-original": "./node_modules/@uniswap/sdk"
```

This won't work for deps installation from clean slate. You need to set up resolutions for TS + webpack instead, without touching deps, or import from node_modules directly.

---

Hi Dima! 👋

---

@anxolin something like:
```json
"resolutions": {
  "@uniswap/sdk-original": "@uniswap/sdk",
  },
```

correct me if I'm wrong obviously

---

Contrary to what you'd expect, `resolutions` don't rename dependencies.
Try 
```suggestion
    "@uniswap/sdk-original": "npm:@uniswap/sdk"
```

---

Hei Dima! sorry I've neglected this PR, we have been very busy. Thanks for dropping by this PR, we miss u man! I hope u are good. Come one day to one zoom call to say hi

I'm re-reviewing this solution. I also saw other alternatives if this one ends up being problematic. I'll also try your solution!

---

Nice trick, `npm:` I struggled a bit but now it's working!